server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -759,13 +759,16 @@ private void fetchScans() {
 
   private void fetchCompactions() {
     ServerContext context = getContext();
+
     for (String server : context.instanceOperations().getTabletServers()) {
       final HostAndPort parsedServer = HostAndPort.fromString(server);
       Client tserver = null;
       try {
         tserver = ThriftUtil.getClient(ThriftClientTypes.TABLET_SERVER, parsedServer, context);
-        var compacts = tserver.getActiveCompactions(null, context.rpcCreds());
-        allCompactions.put(parsedServer, new CompactionStats(compacts));
+        // ELASTICITY_TODO tservers no longer have any compaction information, following code was
```

Comment:
Need to create a follow-on issue, if one does not exist, to figure out how to display compaction information now. There is a page for external compactions, but it should likely be promoted to a more prominent position. It also probably needs some summarization as to how many compactions are occurring per table.

Reply:
I left a comment on a similar issue - but we should look at tracking property, metrics and status info changes so that we can map previous capabilities -> new.  Maybe its start a section in the release notes?

Reply:
> I left a comment on a similar issue - but we should look at tracking property, metrics and status info changes so that we can map previous capabilities -> new. Maybe its start a section in the release notes?

One way to do that would be to open specific issues in the GH elasticity project.  Something like this "Change X needs user facing documentation".

Reply:
> Need to create a follow-on issue, if one does not exist, to figure out how to display compaction information now.

I created #3594 and #3608, could add a comment to those about anything you think needs to be done.  Or open a new issue and link to those.